charger/ocpp.go

```diff
@@ -438,18 +494,19 @@ func (c *OCPP) getCurrent() (float64, error) {
 
 		if err == nil {
 			if resp.ChargingSchedule != nil && len(resp.ChargingSchedule.ChargingSchedulePeriod) > 0 {
-				current = resp.ChargingSchedule.ChargingSchedulePeriod[0].Limit
+				// return first (current) period limit
+				limit = resp.ChargingSchedule.ChargingSchedulePeriod[0].Limit
 			} else {
 				err = fmt.Errorf("invalid ChargingSchedule")
 			}
 		}
 
 		rc <- err
-	}, c.conn.ID(), 1)
+	}, c.conn.ID(), duration)
 
 	err = c.wait(err, rc)
 
-	return current, err
+	return limit, err
```

Comment:
Spräche hier hier etwas gegen
```suggestion
	return limit, c.wait(err, rc)
```
?

Reply:
Inhaltlich nicht. Wir haben sonst bei den Dingen die "vorher" passieren immer eine zweite Zeile spendiert. Funktionieren sollte es.